src/main/resources/META-INF/resources/primefaces/ai/smartcam/smartcam.js

```diff
@@ -0,0 +1,122 @@
+PrimeFaces.widget.SmartCam = PrimeFaces.widget.BaseWidget.extend({
+	
+	init: function(cfg) {
+    	this._super(cfg);
+        if(this.cfg.disabled) {
+            return;
+        }
+        
+        this.cfg.width = this.cfg.width || 320;
+        this.cfg.height = this.cfg.height || 240;
+        this.cfg.renderTimeout = this.cfg.renderTimeout || 50;
+        
+        if (!("autoStart" in this.cfg)) {
+            this.cfg.autoStart = true;
+        }
+        
+        this.modelLoader = this.cfg.model || this.defaultLoad;
+        this.doPrediction = this.cfg.predict || this.defaultPredict;
+
+        if(this.modelLoader && this.doPrediction) {
+        	this.initializeDetectionModel();
+        }
+        
+        if(this.cfg.imageHandler) {
+        	this.imageHandler = this.cfg.imageHandler;
+        } else {
+        	this.imageHandler = function(video, canvasContext, detections) {
+        		canvasContext.drawImage(video, 0, 0, video.width, video.height);
+        	}
+        }
+        
+        this.loop();
+        
+        if (this.cfg.autoStart) {
+            this.play();
+        }
+
+    },
+    
+	defaultLoad: async function() {
+		return await facemesh.load();
```

Comment:
`await` makes this not work in IE.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await